### DISCUSSION TITLE
Great work! How to donate money to the project?

### DISCUSSION THREAD

**Discussion by roburban** [xournalpp/xournalpp#3855] [General] [answered]
Hello,

thank you very much for this great tool.

I would like to make a financial contribution to the project, but I can't find any information on how to do this. I live in Germany and I can easily initiate a bank-transfer to any bank within the EU, or I can write a check on a US bank and send it via snail-mail.

Please let me know the best way.

Regards,
rob

**Comment by Technius** [comment]
Hi @roburban, currently we do not accept financial contributions, but the maintenance team greatly appreciates your kind gesture!

**Reply by bittner** [reply]
I just wanted to second what @roburban said. THANK YOU all for this great application. It provides PDF editing for free on Linux and allows signing with a pen on the virtual paper. No printing needed. This is really, really helpful! THANK YOU so much!  :heart: :heart: :heart:

**Comment by personalizedrefrigerator** [comment, marked as answer]
While Xournal++ doesn't accept financial contributions, we use some libraries that do:
 * Our GUI uses [`GTK`](https://www.gtk.org/), which is supported by the `GNOME` fondation. [`GNOME` accepts donations.](https://www.gnome.org/donate/) 
 * Our plugins are written in the `lua` programming language. [The `Lua` project accepts donations](https://www.lua.org/donations.html).
 * Freedesktop.org provides hosting for some of the projects we use. [They accept donations here.](https://wiki.freedesktop.org/www/#donations)
     * [Poppler](https://poppler.freedesktop.org/) is used for PDF rendering/export.
     * [Cairo](https://www.cairographics.org/) is one of the libraries we use for rendering.
 * We use other open source libraries/technologies that may/may not be accepting donations.
